pubsub/aws/snssqs/snssqs.go

```diff
@@ -65,6 +65,27 @@ type snsSqsMetadata struct {
 	messageMaxNumber int64
 }
 
+type ArnEquals struct {
```

Comment:
Can this be made a non exported field? (ie. `arnEquals instead of ArnEquals`).

This comment is also true for `Condition` and `Statement`.

Reply:
I don't think it is technically possible as the JSON unmarshalling needs the fields to be public. but maybe I'm wrong. see this: https://stackoverflow.com/a/11129474

Reply:
The fields can be exported, that's fine - I'm referring to the struct itself.

Reply:
👍🏿